osx/post_installation.sh

```diff
@@ -90,7 +100,7 @@ function create_plist {
     echo -e "\t<array>" >> $PLIST_SRC
     echo -e "\t\t<string>sh</string>" >> $PLIST_SRC
     echo -e "\t\t<string>-c</string>" >> $PLIST_SRC
-    echo -e "\t\t<string>launchctl setenv KALITE_PYTHON \"$PYRUN\"</string>" >> $PLIST_SRC
+    echo -e "\t\t<string>launchctl setenv KALITE_PYTHON \"$PYTHON\"</string>" >> $PLIST_SRC
```

Comment:
This will be useful when the Mac is restarted because the `KALITE_*` env vars will then be available.